lxd/devices.go

```diff
@@ -431,7 +431,7 @@ func deviceTaskSchedulerTrigger(srcType string, srcName string, srcStatus string
 	}()
 }
 
-func deviceIsDevice(path string) bool {
+func fileIsBlockdev(path string) bool {
```

Comment:
Can we call this deviceIsBlockdev instead?

I've been trying (with limited success) to namespace the functions by their filename, so functions coming from devices.go are all called deviceXYZ

That's so I can track them down easier and also make easier progress at splitting some of that stuff into new golang packages.